src/Container/Container.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ComponentPropsWithoutRef, CSSProperties } from 'react';
+import styles from './Container.module.css';
+import cn from 'classnames';
+
+export interface ContainerProps extends ComponentPropsWithoutRef<'div'> {
+  /**
+   * Container의 min-width값입니다.
+   */
+  minWidth?: string;
+  /**
+   * Container의 max-width값입니다.
+   * @default '100%'
+   */
+  maxWidth?: string;
+  /**
+   * Container의 자식 컴포넌트들의 중앙정렬 여부입니다.
+   * @default false
+   */
+  centerContent?: boolean;
+  /**
+   * Container의 배경색입니다.
+   * @default 'white'
+   */
+  backgroundColor?: CSSProperties['color'];
+  /**
+   * Container의 border-radius값입니다.
+   */
+  borderRadius?: string | number;
+  /**
+   * Container의 padding값입니다.
+   */
+  padding?: string | number;
+}
+
+const Container = ({
+  minWidth,
+  maxWidth = '100%',
+
+  centerContent = false,
+  backgroundColor = 'white',
+  borderRadius,
+  padding,
+  children,
+  ...props
+}: ContainerProps) => {
+  const containerStyle = {
+    minWidth: typeof minWidth === 'number' ? `${minWidth}px` : minWidth,
+    maxWidth: typeof maxWidth === 'number' ? `${maxWidth}px` : maxWidth,
```

Comment:
```suggestion
    minWidth: minWidth !== undefined ? minWidth : 'fit-content' ,
    maxWidth: maxWidth !== undefined ? maxWidth : '100%,
```

`minWidth`와 `maxWidth`의 타입에는 `string | undefined`가 올 수 있는데 `containerStyle`을 만들 때는 해당 props들의 타입이 number인지를 검사하고 있네요! 지금 타입에서는 위와 같이 작성하는 것도 괜찮을 것 같습니다. 아니면 사용성을 위해 number 타입을 추가하는 것도 좋을 것 같아요!